### PR TITLE
Added multiple pages management

### PR DESCRIPTION
**Pull request by ArnaudGathy**

### CODE REVIEW

File: src/SpreadsheetReader.ts [hunk 1]
```diff
@@ -78,39 +113,53 @@ export class SpreadsheedReader {
...
 		try {
-			if( ! this.spreadsheetsIs ) {
+			if( ! this.spreadsheetsId ) {
 				throw Error('Invalid spreadsheetsIs')
 			}
-			const url = `https://spreadsheets.google.com/feeds/cells/${this.spreadsheetsIs}/1/public/full?alt=json`
-			const request = new Request(url, { method: 'GET', contentType: 'application/json'})
 
-			return this.processSpreadsheets(await this.httpClient.execute(request))
+			// We don't want to run all the requests everytime
```
Comment (lmancel) [inline src/SpreadsheetReader.ts:145]: What's the recommended way to retrieve potential updates done to the spreadsheet's content then ?

File: src/SpreadsheetReader.ts [hunk 2]
```diff
@@ -78,39 +113,53 @@ export class SpreadsheedReader {
...
 
 	/**
 	 * Load spreadsheet data
 	 */
-	async loadSpreadsheetData(): Promise<GoogleJsonSpreadsheet> {
+	async loadSpreadsheetData(): Promise<void> {
 		try {
-			if( ! this.spreadsheetsIs ) {
+			if( ! this.spreadsheetsId ) {
 				throw Error('Invalid spreadsheetsIs')
```
Comment (lmancel) [inline src/SpreadsheetReader.ts]:
```suggestion
				throw Error('Invalid spreadsheetsId')
```

**Review by lmancel — CHANGES_REQUESTED** [review]

File: src/SpreadsheetReader.ts (hunk 1 above)
Comment (ArnaudGathy) [reply src/SpreadsheetReader.ts:145]: A new instance of SpreadsheetReader is created everytime we open a modal in the admin, so all the data will be updated when we first call loadSpreadsheetData(). I don't think there is a use case where we want to update the content of the sheet while the modal is open (so updating the instance's data)

Comment (lmancel) [reply src/SpreadsheetReader.ts:145]: I was also worrying about the usage of this module in flowr-frontend, but indeed a new instance of the reader is created each time.

**Pull request merged**